_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,309 @@
+      creating an asymmetric exchange rate risk.  By comparison, when
+      transaction replacement isn't possible, Alice and Bob share the
+      same exchange rate risk.
+
+        Greg Sanders and Jeremy Rubin [replied][sanders cpfp] in
+        [separate][rubin cpfp] emails to note that merchant Bob could
+        incentivize miners to confirm customer Alice's original
+        transaction using [CPFP][topic cpfp], particularly if [package
+        relay][topic package relay] was enabled.
+
+        Antoine Riard [noted][riard free option] that the same risk
+        exists with LN, as Alice could wait to pay merchant Bob's
+        invoice up until shortly before it expired, giving her time to
+        wait for the exchange rate to change.  Although in that case, if
+        Bob noticed that the exchange rate had changed significantly, he
+        could instruct his node not to accept the payment, returning the
+        money to Alice.
+
+    - *Bitcoin Core not in charge of network:* Gloria Zhao [wrote][zhao
+      no control] in the IRC discussion, "I think whatever option we
+      take, it should be made abundantly clear to users that Core does
+      not control whether full RBF happens or not. We could revert
+      [25353][bitcoin core #25353] and it could still happen. [...]"
+
+    - *No removal means problem could happen:* in the IRC discussion,
+      Anthony Towns [echoed][towns uncoordinated] his points from last
+      week, "if we're not going to remove the mempoolfullrbf option from
+      24.0, we're going for an uncoordinated deployment."
+
+        Greg Sanders was [doubtful][sanders doubt], "the question is
+        will 5%+ set a variable. I suspect not."  Towns [replied][towns
+        uasf], "[UASF][topic soft fork activation] `uacomment`
+        demonstrated it's easy to get ~11% to set a variable in just a
+        couple of weeks".
+
+    - *Should be an option:* Martin Zumsande [said][zumsande option] in
+      the IRC discussion, "I think that if a meaningful number of node
+      operators and miners want a specific policy, it shouldn't be on
+      the devs to tell them 'you can't have that now'. Devs can and
+      should give a recommendation (by picking the default), but
+      providing options to informed users should never be a problem."
+
+    As of this writing, no clear resolution had been reached.  The
+    `mempoolfullrbf` option is still included in the release candidates
+    for the upcoming version of Bitcoin Core 24.0 and it is Optech's
+    recommendation that any service depending on zero conf transactions
+    carefully evaluate the risks, perhaps starting by reading the emails
+    linked in [last week's newsletter][news222 rbf].
+
+- **CoreDev.tech transcripts:** prior to The Atlanta Bitcoin Conference
+  (TabConf), several developers participated in a CoreDev.tech event.
+  [Transcripts][coredev xs] of several of the meetings from the event
```

Comment:
Couple items that might be useful for the reader regarding the event, include at your discretion:

- 40 developers
- transcripts only represent about half of the meetings that occurred
- meeting topics were chosen by attendees / unconference style